src/encdatavault_fmt_plug.c

```diff
@@ -24,27 +24,32 @@ john_register_one(&fmt_encdadatavault);
 #include "md5.h"
 #include "common.h"
 #include "formats.h"
+#include "pbkdf2_hmac_sha256.h"
 
 #define FORMAT_TAG              "$encdv$"
 #define FORMAT_TAG_LENGTH       (sizeof(FORMAT_TAG) - 1)
 #define FORMAT_LABEL            "ENCDataVault"
 #define FORMAT_NAME             ""
-#define ALGORITHM_NAME          "MD5"
-#define BENCHMARK_COMMENT       " (1000 iterations)"
+#define ALGORITHM_NAME          "MD5 / PBKDF2-SHA256, AES"
+#define BENCHMARK_COMMENT       ""
 #define BENCHMARK_LENGTH        0x107
 #define PLAINTEXT_LENGTH        125
 #define BINARY_SIZE             0
 #define BINARY_ALIGN            1
-#define SALT_SIZE               sizeof(struct custom_salt)
+#define SALT_SIZE               sizeof(struct enc_salt)
 #define SALT_ALIGN              4
+#ifdef SIMD_COEF_32
+#define MIN_KEYS_PER_CRYPT      SSE_GROUP_SZ_SHA256
+#define MAX_KEYS_PER_CRYPT      SSE_GROUP_SZ_SHA256
+#else
 #define MIN_KEYS_PER_CRYPT      1
 #define MAX_KEYS_PER_CRYPT      4
-
+#endif
```

Comment:
The empty line changes in this PR are weird - all(?) look like they hurt rather than improve clarity. I guess some might have been because of editor settings, but probably not the one above. In general, please undo them.